fabtests/pytest/sm2/conftest.py

```diff
@@ -4,8 +4,3 @@
 @pytest.fixture(scope="module", params=["host_to_host"])
 def memory_type(request):
     return request.param
-
-# TODO Restore delivery_complete when completed
-@pytest.fixture(scope="module", params=["transmit_complete"])
-def completion_type(request):
```

Comment:
The commit says it is adding delivery complete testing but it only removes some code. I'm not super familiar with the pytest framework since we don't use it but that seems wrong?

Reply:
Pytest is fantastic, checkout the fixtures documentation!
https://docs.pytest.org/en/6.2.x/fixture.html

I overloaded the fixture `completion_type` in the SM2 directory with transmit_complete only parameterized fixture.  When I remove my override, we go back to using the one defined in the upper directories conftest.py which looks like:

```
@pytest.fixture(scope="module", params=["transmit_complete", "delivery_complete"])
def completion_type(request):
    return request.param
```